### PR TITLE
starter-journal-article:0.1.1

### PR DESCRIPTION
**Pull request by HPDell**
I am submitting
- [x] a new package
- [ ] an update for a package

Description: This package provides a template for writing journal articles
to organise authors, institutions, and information of corresponding authors.

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

File: packages/preview/starter-journal-article/0.1.1/typst.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[package]
+name = "starter-journal-article"
+version = "0.1.1"
+entrypoint = "lib.typ"
+authors = ["HPDell <https://github.com/HPDell>"]
+license = "MIT"
+description = "A starter template for journal articles."
+repository = "https://github.com/HPDell/typst-starter-journal-article"
+exclude = ["*.pdf", "thumbnail.png"]
```
Comment (reknih) [inline packages/preview/starter-journal-article/0.1.1/typst.toml:9]: You could ignore the .gitignore but that's not blocking

File: packages/preview/starter-journal-article/0.1.1/template/main.typ [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#import "@preview/starter-journal-article:0.1.0": article, author-meta
```
Comment (reknih) [inline packages/preview/starter-journal-article/0.1.1/template/main.typ:1]: This says 0.1.0 but the package version is 0.1.1 - that means this import fails

File: packages/preview/starter-journal-article/0.1.1/typst.toml (hunk 1 above)
Comment (HPDell) [reply packages/preview/starter-journal-article/0.1.1/typst.toml]: Thanks. I've added `.gitignore` into `exclude`.

File: packages/preview/starter-journal-article/0.1.1/template/main.typ (hunk 2 above)
Comment (HPDell) [reply packages/preview/starter-journal-article/0.1.1/template/main.typ]: Thanks. This error has been fixed.

**Comment by reknih** [comment]
Thank you!

**Pull request merged**